### PR TITLE
GSOD 2020 blog post, et al.

### LINKED ISSUE #1204: blog: can't develop/build locally on Win
I'm on Windows (Git Bash on Cmder). I cleaned up mode modules, buid, cache, etc. to build from scratch. I ran `yarn` from scratch, which went well. I ran `yarn develop` and it built in a few minutes, no errors. But I get this when I go to http://localhost:8000/blog:

image

(The rest of the site seems to work.)

Then I tried `yarn build` but it throws an error:

image

### PR DESCRIPTION
**Pull request by jorgeorpinel**
For the application to https://developers.google.com/season-of-docs

Also fixes #1204 (thanks @rogermparent)

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Preview [WIP]: https://dvc-landing-gsod-tiadsxlbt44o3.herokuapp.com/blog/gsod-ideas-2020

**Comment by jorgeorpinel** [comment]
Quick questions @andronovhopf:
- How do you make a comments topic, just create it in https://discuss.dvc.org/c/blog-discussions/5 and link in the frontmatter?
- Did you make the front image of https://dvc.org/blog/gsoc-ideas-2020 and do you think we should reuse it? I guess not since "season" is not as summery 🌴 . Any ideas for the image would be appreciated.

**Comment by andronovhopf** [comment]
Hey Jorge!
1. Yes that's exactly how you create a comment. Let me know if you hit any
snags
2. I did make the front image for that. It was a little hard to come up
with a good image, and I'm not sure I have any ideas off the top of my
head. I'll think on it. Maybe something using their logo and ours?

**Comment by shcheklein** [comment]
@jorgeorpinel hey, there is also GSOD post (not summer of code) that has an image that we can potentially just reuse

**Comment by jorgeorpinel** [comment]
> that's exactly how you create a comment

Yeah I can't create topics there it seems. My user needs privileges, or if you can just create it that's also good. Thanks!

OK Ivan, I'll reuse it from here for now: https://dvc.org/blog/dvc-project-ideas-for-google-summer-of-docs-2019

**Comment by andronovhopf** [comment]
Yeah I can create the comment 

+1 for reusing GSOD image

**Comment by jorgeorpinel** [comment]
Please lmk when the topic exists Elle? For now I removed the comments from the post (maybe even that's best for this one).

@shcheklein please check the project ideas I came up with, you probably have some other things on your mind so there aren't final, just a first proposal: https://dvc-landing-gsod-tiadsxlbt44o3.herokuapp.com/blog/gsod-ideas-2020#project-ideas

**Comment by andronovhopf** [comment]
@jorgeorpinel here's the link: https://discuss.dvc.org/t/join-dvc-for-google-season-of-docs-2020/375

File: content/blog/2020-04-30-gsod-ideas-2020.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+   embedded into the corresponding pages per
+   [this issue](https://github.com/iterative/dvc.org/issues/670).
+
+   This may involve working with our front-end team or, preferably, having some
+   Javascript coding experience.
+
+   _Difficulty rating:_ Medium-Advanced
+
+1. **Javascript engine UI/UX.** Our website has custom
+
```
Comment (shcheklein) [inline content/blog/2020-04-30-gsod-ideas-2020.md:103]: no space needed?

Comment (jorgeorpinel) [reply content/blog/2020-04-30-gsod-ideas-2020.md:102]: fixed.

**Pull request merged**